code_of_conduct.md

```diff
@@ -0,0 +1,5 @@
+## Kubernetes Community Code of Conduct
```

Comment:
Since it's a project-wide CoC I thought `Kubernetes` would be enough, but I can personalise if you think it's better :)